models/apilog.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+module.exports = (sequelize,DataTypes) => {
+
+	var Apilog = sequelize.define('apilog',{
+		rowid: {
+			autoIncrement: true,
+			type: DataTypes.INTEGER(11),
+			primaryKey: true
+		},
+		caller: DataTypes.STRING(10),
+		bloodgroup: DataTypes.STRING(8),
+		stdcode: DataTypes.STRING(6),
+		pincode: DataTypes.STRING(6),
```

Comment:
I think sequelize supports awesome validation and we should use that feature here. Will help us avoid code on the API layer. All the properties should have validators added. The most effective way of doing this would be to create a common validators file with functions that will be common across tables.

Reply:
If we do this step, the problem of the server returning 500 would never come.